### PR TITLE
New controllers, add documentation and a "Playground"

### PR DESCRIPTION
**Pull request by fedegl**
#### Introduce 2 new stimulus controllers
- ChartController
- TagsController

#### Other things
- Document a few of the existing controllers
- Create a "playground" to develop individual components
- Update the RemoteModalController to use stimulus targets

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -37,10 +37,8 @@ GEM
     loofah (2.9.0)
       crass (~> 1.0.2)
       nokogiri (>= 1.5.9)
-    mini_portile2 (2.5.0)
-    minitest (5.14.3)
-    nokogiri (1.11.1)
-      mini_portile2 (~> 2.5.0)
+    minitest (5.14.4)
+    nokogiri (1.11.1-x86_64-darwin)
```
Comment (rafaelcdelgado) [inline Gemfile.lock:41]: On the line 39, there is the same Library with different version. Does it work for multi Environments or we have to have those 2?

File: README.md [hunk 2]
```diff
@@ -1,39 +1,84 @@
...
+import { Application } from 'stimulus'
+import { DropdownController } from 'frontend-helpers'
+
+const application = Application.start()
+application.register('dropdown', DropdownController)
+```
+
+### Development
+
+The javascript code live under the `javascript` folder.
```
Comment (rafaelcdelgado) [inline README.md:51]: Remember to modify, the Code on the index.scss for the node_modules/bulma/bulma.scss for the App Look on the Relative URL "./"   Not on the Absolute Path.

File: app/assets/stylesheets/fe_variables.scss [hunk 3]
```diff
@@ -7,9 +7,32 @@ $size-6: 1rem;
...
+$orange: hsl(14, 100%, 53%);
+$yellow: hsl(48, 100%, 67%);
+$green: hsl(141, 53%, 53%);
+$turquoise: hsl(171, 100%, 41%);
+// $cyan:         hsl(204, 71%,  53%);
+$blue: hsl(217, 71%, 53%);
+$purple: hsl(271, 100%, 71%);
+$red: hsl(348, 86%, 61%);
+
+$afal-blue: #00b3ee;
```
Comment (rafaelcdelgado) [inline app/assets/stylesheets/fe_variables.scss:35]: Lets keep working with the Nomenclature of the web Colors... Use hsl(195, 100%, 47%) instead Hex

File: javascript/src/controllers/tags-controller.js [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+
+    this.selectedItems.forEach(i => this.renderSelectedItem(i))
+  }
+
+  windowResize () {
+    this.hideResults()
+  }
+
+  onKeydown (event) {
+    switch (event.key) {
```
Comment (rafaelcdelgado) [inline javascript/src/controllers/tags-controller.js:52]: Missing the Backspace (delete) tags.

**Review by rafaelcdelgado — CHANGES_REQUESTED** [review]
Just wondering, if the TagsController works fine right now...

It works on the Rendering, but not sure if the request will work. Will test! 

Thanks man

File: Gemfile.lock (hunk 1 above)
Comment (fedegl) [reply Gemfile.lock:41]: This is the actual dependency installed: nokogiri (1.11.1-x86_64-darwin). The other 2 lines are nested under another library and have a range of acceptable versions. Line 39 has this: `nokogiri (>= 1.5.9)` which means that the `loofah` gem accepts any nokogiri version equal to or greater than 1.5.9.  In this case we have 1.11.1, which means it falls within the acceptable range.

File: app/assets/stylesheets/fe_variables.scss [hunk 5]
```diff
@@ -7,9 +7,32 @@ $size-6: 1rem;
...
 
 $primary: $afal-blue;
 $body-background-color: #f5f6f8;
```
[the remark this answers is not in the document]
Comment (fedegl) [reply app/assets/stylesheets/fe_variables.scss:38]: What are you using to transforming the color values?

**Comment by fedegl** [comment]
@rafaelcdelgado Ya quedaron los cambios! Gracias por tus comentarios 👍

**Review by rafaelcdelgado — APPROVED** [review]

**Pull request merged**